client/app/bundles/course/lesson_plan/items.jsx

```diff
@@ -1,10 +1,38 @@
+import React from 'react';
 import { render } from 'react-dom';
-import lessonPlanPlugin from './startup/LessonPlanPlugin';
+import { Provider } from 'react-redux';
+import { IntlProvider, addLocaleData } from 'react-intl';
+
+import zh from 'react-intl/locale-data/zh';
+
+import createStore from './store';
+import LessonPlanContainer from './containers/LessonPlanContainer';
+import translations from '../../../../build/locales/locales.json';
+
+function renderLessonPlan(props) {
+  const i18nLocale = $("meta[name='server-context']").data('i18n-locale');
```

Comment:
Yep, I think that's way to go. We should factor out the Intl logic to a separate container. I will defer this when implementing the next component.